octoprint_octorelay/__init__.py

```diff
@@ -226,15 +228,14 @@ def has_switch_permission(self):
             return False
 
     def on_api_command(self, command, data):
-        self._logger.debug("on_api_command {}, some_parameter is {}".format(command,data))
+        self._logger.debug("on_api_command {}, some_parameter is {}".format(command, data))
 
         # added api command to get led status
         if command == "listAllStatus":
             GPIO.setwarnings(False)
             activeRelays = []
             for key in self.get_settings_defaults():
-                settings = self.get_settings_defaults()[key]
-                settings.update(self._settings.get([key]))
+                settings = self._settings.get([key], merged=True)
```

Comment:
now it's time for me to figure out what `merged=True` does...

Reply:
ok, it seems that this feature only affects the requests having more than 1 item in the setting path array

https://github.com/OctoPrint/OctoPrint/blob/f027aaa7300f70b046525a49245b33c43dd198c1/src/octoprint/settings/__init__.py#L1647-L1650

Reply:
tested the consistency in 77343f4c09c8d72efc4857c06b2043fa9a5cfbf4